src/config/routes.ts

```diff
@@ -14,8 +13,10 @@ export default {
     return basePath
   },
   channel: (id = ':id') => `/channel/${id}`,
-  videos: () => '/videos',
   channels: () => '/channels',
+  video: (id = ':id') => `/video/${id}`,
+  videos: () => '/videos',
+  myVideos: () => '/my-videos',
```

Comment:
This needs to be under `/studio` router, will need rebase. Also probably `/studio/videos` should be enough